cfpb/consumerfinance.gov
# Adjust SVG importing

SVG module export was getting injected into the notification icons.

## Changes

- Changes the inline svg import syntax to use webpack-config existing SVG loader test.
- Changes SVG requires to imports. 


## How to test this PR

1. Visit /blog in Safari and enter some erroneous text for the date and apply the filter. There should be an error message that does not contain the text `module.exports = "`.


## Screenshots

Before:
![Screen Shot 2020-12-22 at 12 24 02 PM](https://user-images.githubusercontent.com/704760/102938306-12615680-447a-11eb-834b-ce4c169fffed.png)
